snuba/consumers/consumer_builder.py

```diff
@@ -375,6 +375,15 @@ def build_dlq_consumer(
             dlq_policy,
         )
 
+    def build_lw_deletions_consumer(
+        self, strategy_factory: ProcessingStrategyFactory[KafkaPayload]
+    ) -> StreamProcessor[KafkaPayload]:
+        return self.__build_consumer(
+            strategy_factory,
+            self.raw_topic,
+            self.__build_default_dlq_policy(),
```

Comment:
Since we don't pass in a `dlq_topic` into the consumer config there is now `dlq_policy` and this doesn't do anything. we can add a dlq later if we want